core/src/test/java/hudson/ClassicPluginStrategyTest.java

```diff
@@ -38,20 +39,20 @@
 
     @Test
     public void test_getDetachedPlugins() {
-        List<DetachedPlugin> list = ClassicPluginStrategy.getDetachedPlugins(new VersionNumber("1.296"));
+        List<DetachedPlugin> list = DetachedPluginsManager.getDetachedPlugins(new VersionNumber("1.296"));
```

Comment:
Then perhaps just move this to a test suite named after the class it is testing. Or at a minimum use `@For`.